core/client/fs/src/main/java/alluxio/client/ReadType.java

```diff
@@ -82,4 +84,15 @@ public boolean isCache() {
   public boolean isPromote() {
     return mValue == CACHE_PROMOTE.mValue;
   }
+
+  // TODO(ggezer) Eliminate below parsers/converters for ReadType
+  private static final String sProtoNameHeader = "READ_";
```

Comment:
why is this needed?

Reply:
gRPC enums are defined in a global scope. Defining them without some kind of header invites conflicts for commonly used enum values. (E.g NONE,ALL, etc...)

Reply:
PS: I don't intend to keep this header trick for longer. Once we start eliminating wire types, they should go away naturally.